### ISSUE TITLE
Lint requiring the file end with one newline

### ISSUE THREAD

**Issue by SamWilsn** [ethereum/eipw#9]
requested by @pandapip1

**Comment by JEAlfonsoP** [comment]
@Pandapip1 or this one, sorry to bother you,  I am working on the eipw-bot this weekend, and just want to try to make some progress. Thanks again.

**Comment by Pandapip1** [comment]
Yup, this is something that would be nice. I bet it's possible to do automatically with gitattributes, though.

**Comment by JEAlfonsoP** [comment]
Is this issue still valid ? so I can implement it.  Please confirm.  Ty.

**Comment by JEAlfonsoP** [comment]
@Pandapip1 please could you comment about it ? Ty

**Comment by Pandapip1** [comment]
Yes, we would like to add a .gitattributes file to the EIPs repository.

**Comment by JEAlfonsoP** [comment]
@SamWilsn I believe this one is still on discussion. ?

EIPW could solves it. Let me know the decision about it. Ty

**Comment by Pandapip1** [comment]
I would strongly prefer if this were solved natively using gitattributes
